db/migrate/20230608001833_add_user_ref_to_like.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserRefToLike < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :likes, :user, null: false, foreign_key: true
```

Comment:
If you check the diagram provided, you'd see authorId on most of the models and that should serve as the foreign key field in the table. So in order to implement the ERD correctly, consider using this syntax here:

```
`t.references :author, foreign_key: { to_table: 'users' }`
```